.github/workflows/ci.yaml

```diff
@@ -315,7 +303,7 @@ jobs:
     needs:
       - arm
       - docker-arm
-    if: ${{ always() && startsWith(github.ref, 'refs/tags/v') }}
+    if: ${{ always() && vars.SSH_ARM_ENABLED) }}
```

Comment:
If I understood correctly, `always()` allows to run this job even after `arm` or `docker-arm` succeeds, fails or are cancelled/skipped. So it only needs to add a condition to run when at least the `arm` job runs, which is the `SSH_ARM_ENABLED` variable.